docs/tutorials/add-oauth2-provider.md

```diff
@@ -181,7 +181,7 @@ After you finished adding your new provider to Appwrite, you should be able to s
 
 Add credentials and check both a successful and a failed login (where the user denies integration on the provider page).
 
-You can test your OAuth2 provider by trying to login using the [OAuth2 method](https://appwrite.io/docs/client/account#accountCreateOAuth2Session) when integrating the Appwrite Web SDK in a demo app.
+You can test your OAuth2 provider by trying to login using the [OAuth2 method](/docs/client/account#accountCreateOAuth2Session) when integrating the Appwrite Web SDK in a demo app.
```

Comment:
we need to revert this change. The tutorials aren't really part of the website. So we cannot use relative links here